src/components/internal/Ellipsis/Ellipsis.tsx

```diff
@@ -1,8 +1,9 @@
 import styled from 'styled-components';
 
 export const Ellipsis = styled.span`
-  display: block;
-  overflow: hidden;
   text-overflow: ellipsis;
-  white-space: nowrap;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
```

Comment:
The `<Ellipsis />` component wants to be generic, so it is ideal to receive line numbers in props. (e.g. `<Ellipsis line={2} />`) 

But I'll make this change after the merge!
Thanks for the great features!